Examples/VisionExamples/Examples/ARNavigationViewController.swift

```diff
@@ -26,9 +26,9 @@ class ARNavigationViewController: UIViewController {
 
         // create VisionManager with video source
         visionManager = VisionManager.create(videoSource: videoSource)
-        // create VisionARManager and register as its delegate to receive AR related events
+        // create VisionARManager
         visionARManager = VisionARManager.create(visionManager: visionManager)
-
+        /// configure AR view to display AR navigation
```

Comment:
@IodaMikeMapbox do we want to use a doc comment or this is a typo?

Reply:
ah, right, it's a typo